images/jdk/configs/openjdk-11.apko.yaml

```diff
@@ -9,6 +9,7 @@ contents:
     - glibc-locale-en
     - busybox
     - openjdk-11
```

Comment:
based on what I heard from @kaniini we just need the openjdk-11-default-jvm package as it pulls in openjdk-11 as a runtime dep.

Reply:
actually I think I misunderstood. hold off on that change

Reply:
I think my understanding now is that -default-jvm only pulls in the JRE and not the JDK. So perhaps we keep it here but remove from the JRE image?  I guess it would have no impact either way because it's a subset or equivalent?